doc/nrf/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield.rst

```diff
@@ -0,0 +1,88 @@
+.. _peripheral_sensor_node_shield:
+
+Peripheral Sensor node Shield
+#############################
+
```

Comment:
Also, shouldn't we mention the PSN sample in this article?

Reply:
It seems to me that in the other boards documentation there is no information about what applications or samples use them.

Reply:
I think it's mostly because other shields can be used in multiple configurations, as far as I understand this shield is currently only applicable to Peripheral sensor node. 
The first sentence suggest that it's more of a general extension to nRF54L15.
Maybe it wouldn't be a bad idea to just add:
``The Peripheral Sensor node shield is an extension for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`, used to develop the Peripheral Sensor node application. `` < reference > (?)

not adding the reference as it won't work until the other doc is merged, plus it will break the build, but that's the general idea. What do you think?

Reply:
Thanks for the explanation, let's leave as is then :)